dev/io.openliberty.faces.4.0.internal/src/io/openliberty/faces40/internal/fips/FIPSInitializer.java

```diff
@@ -0,0 +1,67 @@
+/*******************************************************************************
+ * Copyright (c) 2025 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package io.openliberty.faces40.internal.fips;
+
+import jakarta.servlet.ServletContainerInitializer;
+import jakarta.servlet.ServletContext;
+import jakarta.servlet.ServletException;
+
+import java.util.Map;
+import java.util.ServiceLoader;
+import java.util.Set;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.ibm.ws.common.crypto.CryptoUtils;
+
+public class FIPSInitializer implements ServletContainerInitializer {
+
+    // Log instance for this class
+    protected static final Logger log = Logger.getLogger("io.openliberty.faces40.internal.fips");
+    private static final String CLASS_NAME = "FIPSInitializer";
+
+    private static String MAC_ALGORITHM = "org.apache.myfaces.MAC_ALGORITHM";
+    private static String SESSION_ALGORITHM = "org.apache.myfaces.ALGORITHM";
+    private static String VIEWSTATE_ID_ALGORITHM = "org.apache.myfaces.RANDOM_KEY_IN_VIEW_STATE_SESSION_TOKEN_SECURE_RANDOM_ALGORITHM";
+     // Note 'ALGORITM' is misspelled, but that's how it's documented.
+    private static String CSRF_SESSION_ALGORITHM = "org.apache.myfaces.RANDOM_KEY_IN_CSRF_SESSION_TOKEN_SECURE_RANDOM_ALGORITM";
+
+    @Override
+    public void onStartup(Set<Class<?>> clazzes, ServletContext servletContext) throws ServletException {
+
+        if (CryptoUtils.isFips140_3EnabledWithBetaGuard()) {
+            // **** Already set as default *****
+            // if (servletContext.getInitParameter(MAC_ALGORITHM) == null) {
+            // servletContext.setInitParameter(MAC_ALGORITHM, "HmacSHA256");
+            // log(MAC_ALGORITHM + " not found. Setting to HmacSHA256.");
+            // }
+            // if (servletContext.getInitParameter(SESSION_ALGORITHM) == null) {
+            // servletContext.setInitParameter(SESSION_ALGORITHM, "AES");
+            // log(SESSION_ALGORITHM + " not found. Setting to AES.");
+            // }
```

Comment:
I'd prefer to remove this code and just add a comment stating the following is already set as default so no need to update it for FIPs.